cmd/plasmacli/subcmd/tx/sign.go

```diff
@@ -92,31 +83,22 @@ Usage:
 }
 
 // generate confirmation signature for specified owner and position
-func signSingleConfirmSig(ctx context.CLIContext, position plasma.Position, signerAddr, owner ethcmn.Address, name string) error {
-	// retrieve the new output
-	utxo := store.UTXO{}
-	key := append(owner.Bytes(), position.Bytes()...)
-	res, err := ctx.QueryStore(key, "utxo")
+// verify that the inputs provided are correct
+// signing address should match one of the input addresses
+// generate confirmation signature for given utxo
```

Comment:
*output

Reply:
some of the other helper functions have named return values. For consistency can we remove named return values and document the multiple return values in the comment

Reply:
I'd argue if you have to declare a `var` at the top of a func then named return values are appropriate  otherwise sure. Resolving this though since it doesn't relate to a named return value